src/components/Background/index.view.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./Background.scss";
+import { ReactComponent as Wire } from "images/wire.svg";
+import { ReactComponent as Rainbow } from "images/rainbowpath.svg";
+interface BackgroundProps {
+  children?: React.ReactNode;
+}
+
+const Background: React.FC<BackgroundProps> = ({
+  children,
+}: BackgroundProps) => {
+  return (
+    <div>
```

Comment:
This just looks like an empty `div` container. If it doesn't do very much, I would probably remove it.